packages/api/resolvers/types/assortment.js

```diff
@@ -5,7 +5,7 @@ export default {
   async assortmentPaths(obj, { forceLocale }, { localeContext }) {
     return obj.assortmentPaths(forceLocale || localeContext.normalized);
   },
-  async search(obj, query, context) {
```

Comment:
I'd just copy the new "searchProducts" resolver implementation and name it like the old:

async search -> obj.searchProducts
async searchProducts -> obj.searchProducts